compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1570,22 +1571,47 @@ static LogicalResult setRootConfig(
   return failure();
 }
 
-namespace {
-bool is2DPoolingOp(linalg::LinalgOp op) {
-  return isa<linalg::PoolingNhwcSumOp, linalg::PoolingNchwSumOp,
-             linalg::PoolingNhwcMaxOp, linalg::PoolingNhwcMaxUnsignedOp,
-             linalg::PoolingNhwcMinOp, linalg::PoolingNhwcMinUnsignedOp,
-             linalg::PoolingNchwMaxOp>(op.getOperation());
+static bool is2DConvOp(Operation *op) {
+  return isa<linalg::Conv2DNhwcHwcfOp, linalg::Conv2DNchwFchwOp>(op);
+}
+
+static bool is2DDepthConvOp(Operation *op) {
+  return isa<linalg::DepthwiseConv2DNhwcHwcOp>(op);
+}
+
+static bool is2DPoolingOp(Operation *op) {
+  return isa<linalg::PoolingNhwcSumOp, linalg::PoolingNhwcMaxOp,
+             linalg::PoolingNhwcMaxUnsignedOp, linalg::PoolingNhwcMinOp,
+             linalg::PoolingNhwcMinUnsignedOp, linalg::PoolingNchwSumOp,
+             linalg::PoolingNchwMaxOp>(op);
+}
+
+/// Helper enum to represent conv2d input traversal order.
+enum class Conv2DOpOrder {
```

Comment:
-> DimOrder?

Reply:
I followed upstream convention, but I agreed that `DimOrder` looks better. changed!